x/superfluid/keeper/gov/gov.go

```diff
@@ -40,7 +37,7 @@ func HandleRemoveSuperfluidAssetsProposal(ctx sdk.Context, k keeper.Keeper, p *t
 			return err
 		}
 		dummyAsset := types.SuperfluidAsset{}
-		if asset.Equal(dummyAsset) {
+		if asset == dummyAsset {
```

Comment:
This is confusing to me, shouldn't it have been .Equal( all along?

Reply:
This was `==` all along, I remember when reviewing it, I'll merge here and create a new PR with this change.

Reply:
https://github.com/osmosis-labs/osmosis/pull/8580 

:arrow_double_up: